integration/hive/src/main/java/org/apache/carbondata/hive/MapredCarbonOutputFormat.java

```diff
@@ -61,24 +68,25 @@ public void checkOutputSpecs(FileSystem fileSystem, JobConf jobConf) throws IOEx
   public FileSinkOperator.RecordWriter getHiveRecordWriter(JobConf jc, Path finalOutPath,
       Class<? extends Writable> valueClass, boolean isCompressed, Properties tableProperties,
       Progressable progress) throws IOException {
+    ThreadLocalSessionInfo.setConfigurationToCurrentThread(jc);
     CarbonLoadModel carbonLoadModel = null;
-    String encodedString = jc.get(LOAD_MODEL);
+    // Take carbonLoadModel from container environment if set.
```

Comment:
i think here first we should try to get directly from conf, if we dont get then we can try from systemEnv, it will serve all

Reply:
done

Reply:
done